### DISCUSSION TITLE
Questions about conversion and GUI

### DISCUSSION THREAD

**Discussion by OldPixelReaper** [Amblyopius/Stable-Diffusion-ONNX-FP16#9] [General]
Hello, first of all thank you for your support in the matter of WIN AMD SD. Everything works really well so far, but I have some questions. 

1. I have used your scripts some time ago, but now somehow missing the possibility to edit something else at the step "Diffusers"? Am I doing something wrong or has the conversion recently changed from --.safetensors or .ckpt to Diffusers to Onnx-- to --.safetensors or .ckpt to Onnx--?

I must admit that I have no idea, but I still did this step when converting:

https://huggingface.co/wavymulder/portraitplus/commit/8c72c4b5a7454d32f12479af0054295c34f4f76c

I don't know if that was right, but now I can't do that anymore^^ Sometimes I inserted the VAE there too.(Wrong?)

2. the question about Textual Inversion was answered.

3. is it somehow possible to integrate your run-batch solution into the GUI? I am a GUI person^^ and I test many models. I would find it super nifty if I could say for example, wow, great image, let me see that with all models or pipelines without having to leave the GUI. 

4. since the diffusers step is kinda gone i currently use this script:https://gist.githubusercontent.com/Pyr-000/8513eb989e44d7c4085216fab452d749/raw/2f26a22fe55cb82ce216ad0a035c07558678f6db/enhanced_convert_original_stable_diffusion_to_diffusers.py

Is this still compatible with the new huggingface and ORT updates?

5. sorry for my really bad English :-)

thx and have a nice weekend

**Comment by Amblyopius** [comment]
1) The step was indeed removed in the sense that it supports direct conversion. In the past you needed to do the ckpt -> diffusers step yourself. As not everyone got that working well enough we now integrated it. Though the script in point 4 should be fine and you can then convert from local diffusers to ONNX like you did before.

For the step that you were still doing yourself: I'm currently looking at adding options that automatically fix/alter things at the diffusers point. The main one being clip skip. If the VAE change makes sense it would be possible to add that too.

3) Probably yes, there's little reason why it wouldn't be possible. It might have a tad less functionality but it should be possible at some point to get an extra tab in the UI to generate variations. May need community contribution though as it's not very high on the list things I'd be looking at right now.

4) As discussed in point 1, yes it is. It gives you a local diffusers you can then convert.

5) The quality of your English is more than adequate, don't worry :)

**Reply by OldPixelReaper** [reply]
thanks for the quick reply!
